conda_forge_tick/migrators.xsh

```diff
@@ -600,6 +601,37 @@ class Rebuild(Migrator):
         return n
 
 
+class CompilerRebuild(Rebuild):
+    bump_number = 1000
+
+    def migrate(self, recipe_dir, attrs, **kwargs):
+        with indir(recipe_dir):
+            with indir('..'):
```

Comment:
It seems like you could save an indentation level by doing `with indir(recipe_dir), indir('..'):` or better ye, just `with indir(recipe_dir + '/..'):`